lib/commands/oad.js

```diff
@@ -15,6 +15,14 @@ exports.builder = {
     describe: 'A boolean flag indicating whether output format of the messages is json.',
     boolean: true,
     default: true
+  },
+  o: {
+    alias: 'old-tag-name',
+    describe: 'The tag name for the readme.  If include also indicates that the old spec file is a readme file',
+  },
+  n: {
+    alias: 'new-tag-name',
+    describe: 'The tag name for the readme.  If include also indicates that the old spec file is a readme file',
   }
```

Comment:
also we need to update the readme for the new option

Reply:
Also the version in the package.json

---
In reply to: [189362203](https://github.com/Azure/openapi-diff/pull/115#discussion_r189362203) [](ancestors = 189362203)

Reply:
Done